src/transformers/models/vit_mae/modeling_vit_mae.py

```diff
@@ -948,6 +1051,7 @@ def forward(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
+        interpolate_pos_encoding: Optional[bool] = False,
```

Comment:
nit - we want this flag to be a boolean, and don't need to handle null values 

```suggestion
        interpolate_pos_encoding: bool = False,
```